shared/tests/utils/test_mcp_utils.py

```
@@ -14,10 +15,10 @@ def test_replace_mcp_server_variables_replaces_backend_url_and_task_token():
             "timeout": 300,
         }
     }
-    task_data = {
-        "backend_url": "http://localhost:8000",
-        "task_token": "token-123",
-    }
+    task_data = ExecutionRequest(
+        backend_url="http://localhost:8000",
+        auth_token="token-123",
```

Ruff (0.15.2)

[error] Possible hardcoded password assigned to argument: "auth_token"

(S106)